examples/master/README.md

```diff
@@ -0,0 +1,102 @@
+# Jenkins Master example
+
+This directory contains an example of Jenkins setup, that is configured to
+demonstrate the CI/pipeline workflow for the sample-app application using the
+Jenkins Master/Slave setup and automatization done on OpenShift v3.
+
+<p align="center">
+<img width="420" src="https://raw.githubusercontent.com/mfojtik/jenkins-ci/master/jenkins-flow.png"/>
+</p>
+
+## Installation
+
+To start, you have to manually enter following commands in OpenShift:
+
+
+```console
+# Create project and allow Jenkins to talk to OpenShift API
+$ oc new-project ci
+$ oc policy add-role-to-user edit system:serviceaccount:ci:default
+
+# Create the 'staging' project where we deploy the sample-app for testing
+$ oc new-project stage
+$ oc policy add-role-to-user edit system:serviceaccount:stage:default
+$ oc policy add-role-to-user edit system:serviceaccount:ci:default
+
+# Now create the templates in the 'ci' and 'stage' projects
+$ oc create -f openshift -n ci
+$ oc create -f https://raw.githubusercontent.com/openshift/jenkins/master/examples/master/jenkins-master-with-k8s-plugin.json
+$ oc create -f https://raw.githubusercontent.com/openshift/jenkins/master/examples/slave/s2i-slave-template.json
```

Comment:
@bparees I also need a sample-app with some fake unit test... any tip what repo to use? ;-) (and I will have to add example unit test there)